### DISCUSSION TITLE
Suggestion about new Geomap panel

### DISCUSSION THREAD

**Discussion by mertbakir** [grafana/grafana#37830] [Dashboards, Panels & Visualizations]
First of all it's very good to see a map panel other than old worldmap. I have a few suggestions.

### Better Legend For Categoric Variables:

For continuous data this legend works fine:
![image](https://user-images.githubusercontent.com/43188411/129168391-7cf4654f-e3df-4924-a9b2-3c2ed99483a6.png)

When it comes to discreate or categoric variables, < and + doesn't look good. In reality, I have 2 options here 3 and 4, but I have to set the thresholds like it's continuous. 

![image](https://user-images.githubusercontent.com/43188411/129168624-d13b5643-771e-4b5c-b519-863da1dc95e0.png)

What I suggest is let's not use thresholds here and set the marker color in marker color section. There should be a toggle between categoric/cont. variables. So, I should be able to set 3 = Green, 4 = Red or 3-4 Green, 4+ Red.

![image](https://user-images.githubusercontent.com/43188411/129169201-80e21b27-67d4-4d95-8605-367eaaea64a5.png)

Also, if possible, marker shape for different categories.
e.g: 3: Triangle, 4: Cross.

In the end I should be able to combine these categoric features with the existing value mapping:

![image](https://user-images.githubusercontent.com/43188411/129169579-bbc40ccd-5f9b-45fa-87e3-5547537dabdc.png)

And have a legend like

not this:
![image](https://user-images.githubusercontent.com/43188411/129169610-d8601957-6d2d-4a71-a7cc-0991ab2f04f9.png)

but this:

![image](https://user-images.githubusercontent.com/43188411/129169926-7a3abd0a-a7ad-4d49-9159-bb0b6b5421e2.png)

excuse my paint skillz.

### On Hover:

- Fields on the hover list should be configurable. For example there might be numeric field for coloring or sizing points but we may not want to display it on hover. Similarly, displaying lat long on hover may provide no insight and these fields just make the list cluttered.
- After hover, on click I would like to make the list sticky until I click anywhere else.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
